src/styles.scss

```diff
@@ -33,3 +33,12 @@ body {
 .main-container {
   background: none;
 }
+// Clear the 'X' from the input type 'search' in various browsers.
```

Comment:
What about other browsers, does it work on them as well?

Reply:
Updated the documentation, it does support majority of other browsers. https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-search-cancel-button